### ISSUE TITLE
[Bug Report] - Error in true_divide -> All ResnetV2 Keras Models

### ISSUE THREAD

**Issue by Meywether** [sicara/tf-explain#130]
Dear Sicara,
thanks for this great easy-to-use library.
I do have to mention a bug in all ResnetV2 Models:
Starting with a small modified version of your provided example, tf-explain throws following error by using:

- ResNet50V2
- ResNet101V2
- ResNet152V2

The "V1" Versions are working properly. -> Tested with all three variants!

The script throws following error:

![image](https://user-images.githubusercontent.com/39849054/80940729-ef960b00-8de0-11ea-8278-3c8c32d0791a.png)

Thanks in advance!
Best regards
Meywether

Attached: My version of your provided example:

![image](https://user-images.githubusercontent.com/39849054/80940760-00468100-8de1-11ea-9b47-73803be5ba18.png)

**Comment by RaphaelMeudec** [comment]
@Meywether This might be due to heatmap.min being equal to heatmap.max. You might want to remove the last softmax layer of the Resnet to prevent this from happening.

**Comment by nishantagarwal** [comment]
> @Meywether This might be due to heatmap.min being equal to heatmap.max. You might want to remove the last softmax layer of the Resnet to prevent this from happening.

@RaphaelMeudec  is right here. Even I had the same issue. I resolved it by removing the last layer from my pre-trained model.

**Comment by Meywether** [comment]
Hello @RaphaelMeudec and @nishantagarwal ,
thanks for your response. 

`    model = tf.keras.applications.resnet_v2.ResNet50V2(weights="imagenet", include_top=False) ` - is not working

So I created a second model by using:
`model2 = Model(model.input, model.layers[-2].output)`

model2.summary() shows me that the props layer is gone, but still getting the same error

I thought that `model2 = Model(model.input, model.layers[-1].output)` should be enough but in my case the Dense probs layer was still in my model.

May I ask you to provide me a code snippet ?

Best regards

Meywether

**Comment by Ram-WD** [comment]
I have the same problem as @Meywether  , with a custom keras model with multi output ( segmentation and classification).
